dataset/dictionary/y-events.json5

```
@@ -1052,7 +1068,7 @@
     en: "Bing-Bang Finchball",
     ja: "カラコロ! チュンボール",
     zhCN: "哐哐当当雀雀球",
-    notes: "v3.8 期間限定イベント「涼夏! 楽園? 大秘境!」のアトラクションの1つ",
+    notes: "v3.8 期間限定イベント「涼夏! 楽園? 大秘境!」のアトラクションの1つ、またはv4.8 期間限定イベント",
```

Added space before alphanumeric word. Just a minor fix for consistency.

```suggestion
    notes: "v3.8 期間限定イベント「涼夏! 楽園? 大秘境!」のアトラクションの1つ、または v4.8 期間限定イベント",
```